### PR TITLE
add one error code

### PR DESCRIPTION
**Pull request by liuyu85cn**

### CODE REVIEW

File: src/common/interface/meta.thrift [hunk 1]
```diff
@@ -76,6 +76,7 @@ enum ErrorCode {
     E_SAVE_JOB_FAILURE       = -57,
     E_BALANCER_FAILURE       = -58,
     E_JOB_NOT_FINISHED       = -59,
+    E_TASK_REPORT_OUT_DATE   = -80,
```
Comment (panda-sheep) [inline src/common/interface/meta.thrift:79]: `-80`， Why is it here?
Best to put the error codes in order?

Comment (liuyu85cn) [reply src/common/interface/meta.thrift:79]: because this is an admin error code, and I don't want split admin error code.

**Review by panda-sheep — APPROVED** [review]
Good，LGTM！

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Pull request merged**